autogpt_platform/backend/backend/util/service.py

```diff
@@ -354,6 +355,7 @@ def __start_fastapi(self):
 
     def run(self):
         super().run()
+        sentry_init()
         self.fastapi_app = FastAPI()
```

Comment:
You might want to place it even before super().run

Reply:
okay cool thanks!